packages/dialect-react-ui/components/common/providers/DialectThemeProvider.tsx

```diff
@@ -305,7 +305,7 @@ export const defaultVariables: Record<ThemeType, ThemeValues> &
       toggleBackground: 'dt-bg-white/60',
       toggleBackgroundActive: 'dt-bg-[#528E5B]',
       toggleThumb: 'dt-bg-[#1F1F1F]',
-      notificationBadgeColor: 'dt-bg-red-500',
+      notificationBadgeColor: 'dt-bg-[#6F2AFF] dt-text-white',
```

Comment:
Should `#6F2AFF` become a color variable in `tailwind.config.js`?

Reply:
I thought we don't really use tailwind.config? We can definitely start, haha

Reply:
I do!

Reply:
<img width="495" alt="Screenshot 2022-09-01 at 14 53 59" src="https://user-images.githubusercontent.com/2504771/187907767-e0b4a476-c460-4338-9dcf-d142a3c9e754.png">

Reply:
Alrighty then! Thank you! This is good!

Gonna move them to theme.